docs/config.md

```diff
@@ -662,6 +662,33 @@ then the `shape` option of the `perf_analyzer_flags` option must be specified.
 More information about this can be found in the 
 [Perf Analyzer documentation](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#input-data).
 
+**Note**:
+Perf Analyzer now supports SSL and can be enabled via Model Analyzer configuration file updates.
+
+GRPC example:
+
+```yaml
+model_repository: /path/to/model/repository/
+profile_models:
+  model_1:
+    perf_analyzer_flags:
+        ssl-grpc-root-certifications-file: /path/to/PEM/encoded/server/root/cert
```

Comment:
underscores

Reply:
Reviewed with Matt offline. Documentation states hyphens are required. Updated the rest of the examples to reflect this.